_includes/parse-server/deploying-aws-ubuntu.md

```diff
@@ -0,0 +1,192 @@
+sudo nano postgresql.conf
+```
+Search for `#listen_addresses='localhost'`, uncomment the line and replace `localhost` for `*`
+
+Restart the PostgreSQL server
+```bash
+sudo service postgresql restart
+```
+
+### Setup Parse Server
+
+Create a directory
+```bash
+cd ~
+```
+```bash
+mkdir parse-server
+```
+```bash
+cd parse-server
+```
+
+Run the bash script and follow the instructions, the script have some visual issues and the keys generation doesn't work.
+```bash
+sh <(curl -fsSL https://raw.githubusercontent.com/parse-community/parse-server/master/bootstrap.sh)
+```
+After that, we need to setup the configuration file, use your own `appId`, `masterKey` and `clientKey`, use random strings or some generator tool to create secured keys.
+```bash
+sudo nano -w config.json
+```
+This are the basic options of the config.json file, for the full list you can type `parse-server --help` or refer to the [Full Options Document](http://parseplatform.org/parse-server/api/master/ParseServerOptions.html) for more details.
+```js
+{
+  "appId": "luhYNQXysW6sS8S2s2c8PP9ZpLAf0gMb",
+  "masterKey": "2AQ45kFMM8CL1HRkWc7IX0GwDW1wffnK",
+  "clientKey": "gQP5nYGP1sC8gpRalJ1TGXUgDDHySdXY",
+  "appName": "MyApp",
+  "cloud": "./cloud/main",
+  "databaseURI": "postgres://postgres:mypassword@localhost:5432/postgres"
+}
+```
+
+
+Install Parse Server globally
+
+```bash
+sudo npm install -g parse-server
+```
+
+Start Parse Server using the script command in the config.json
+```bash
+npm start
+```
+or manually with the nohup command and specifying the configuration file, this option will keep the server running even if you close the terminal
+```bash
+nohup parse-server config.json &
+```
+Check the Server is running `http://host-ip-or-domain:1337`
+
+### Setup Parse Dashboard
+
+Install Parse Dashboard globally
+```bash
+sudo npm install -g parse-dashboard
+```
+
+Once installed, you need to configure Parse Dashboard, go to `/usr/lib/node_modules/parse-dashboard/Parse-Dashboard/` and edit the file `parse-dashboard-config.json`
+```bash
+sudo nano -w parse-dashboard-config.json
+```
+This is an example of parse-dashboard.config.json, check the [full list of options](https://github.com/parse-community/parse-dashboard#configuring-parse-dashboard) for more details.
+```js
+{
+{
+  "apps": [
+    {
+      "serverURL": "http://<your-server-IP-or-domain>:1337/parse",
+      "appId": "luhYNQXysW6sS8S2s2c8PP9ZpLAf0gMb",
+      "masterKey": "2AQ45kFMM8CL1HRkWc7IX0GwDW1wffnK",
+      "allowInsecureHTTP": "true",
+      "appName": "MyApp"
+    }
+  ],
+ "users": [
+    {
+      "user":"admin",
+      "pass":"password"
+    }
+  ]
+}
+```
+
+Start Parse Dashboard
+```bash
+parse-dashboard
+```
+or with the nohup command and specifying the configuration file, this option will keep the dashboard running even if you close the terminal
+```bash
+nohup parse-dashboard --dev --config parse-dashboard-config.json &
+```
+
+Check the Dashboard is running `http://host-ip-or-domain:4040`
```

Comment:
> Ensure that Parse Dashboard is running by opening `http://<DOMAIN>:4040`.

Reply:
I will use <IP/DOMAIN> instead of <DOMAIN>, because actually the terminal always show the IPs instead of Domains, I think is good to know that people can use IPs or Domains in the configuration. and if I had to use only one, I will choose <IP>, but I will stick with both. (maybe <IP_or_DOMAIN> or <IP> or <DOMAIN>... 🤔

Reply:
Yes, `IP_OR_DOMAIN` looks readable.